pyomo/bilevel/components.py

```diff
@@ -13,13 +13,17 @@
 from pyomo.core.base.plugin import ModelComponentFactory
 from pyomo.core.base.component import Component
 from pyomo.core.base.block import SimpleBlock
+from pyomo.common.deprecation import deprecated
 
 # TODO: Do we need to have SimpleSubModel and IndexedSubModel classes?
 
 
 @ModelComponentFactory.register("A submodel in a bilevel program")
 class SubModel(SimpleBlock):
 
+    @deprecated("Use of the pyomo.bilevel package is deprecated. Development of bilevel \
+optimization capabilities has been shifted to the Pyomo Adversarial Optimization (PAO) library. \
+Please contact William Hart for further details (wehart@sandia.gov).")
```

Comment:
I believe that backslashes for line continuation within strings is not recommended.  Can we change this to either:
```python
    @deprecated("Use of the pyomo.bilevel package is deprecated. Development of bilevel "
                "optimization capabilities has been shifted to the Pyomo Adversarial "
                "Optimization (PAO) library. Please contact William Hart for further "
                "details (wehart@sandia.gov).")
```
or 
```python
    @deprecated("""Use of the pyomo.bilevel package is deprecated. Development of bilevel
optimization capabilities has been shifted to the Pyomo Adversarial Optimization (PAO) library.
Please contact William Hart for further details (wehart@sandia.gov).""")
```